steps/firebase-app-distribution/0.1.0/step.yml

```diff
@@ -0,0 +1,108 @@
+title: '[Beta]Firebase App Distribution'
```

Comment:
please have an empty space between the texts and the `[Beta]` part as I am not sure if this way the users can find it by searhing for the firebase keyword.

Also, [BETA] might be more noticeable.

Reply:
apply the comment above for all of this texts

Reply:
done